tests/test_choice_element.py

```diff
@@ -0,0 +1,87 @@
+from nicegui import events, ui
+from nicegui.testing import Screen
+
+
+###
+
+# Tests Needed:
+
+# - Radio options should be clickable, and the value should be set to the clicked option
```

Comment:
I think you can remove this explanations. The test function name should describe what is being tested.